lib/tds.js

```diff
@@ -505,14 +540,23 @@ async function generateTdsRuleset (
     const dnrRules = []
     for (const [trackerDomain, trackerEntry] of Object.entries(tds.trackers)) {
         const requestDomains = requestDomainsByTrackerDomain.get(trackerDomain)
+        if (typeof tds.entities[trackerEntry.owner.name] === 'undefined') {
```

Comment:
Not sure that this check is necessary, or why we need to add it now? Could you explain?

Reply:
Because just below it (tries to) deference `tds.entities[trackerEntry.owner.name].domains`, and from testing with locally generated TDS this would sometimes fail.   Perhaps this is cleaned up during the TDS release process, but regardless it should be handled.

Reply:
Fair enough, but could you get rid of the warning and add a comment to mention that sometimes the block list is malformed, so we should take care to skip tracker entries that don't have a corresponding entity entry?